docs/reference/cli/general.md

```diff
@@ -0,0 +1,240 @@
+
+The `pref validate` command allows the caller to validate a server's internal preference value without setting it.
+
+If you want to validate a preference from a namespace other than `server` (eg. a plugin), you'll have to prepend the desired namespace to the prefname.
+
+Examples:
+```
+Request: "pref validate bufferSecs 10"
+Response: "pref validate bufferSecs valid:1"
+
+Request: "pref validate audiodir %2fsome%2fincorrect%2ffilepath"
+Response: "pref validate audiodir valid:0"
+```
+***
+
+## artworkspec
+`artworkspec add <specification> <name>`
+
+The `artworkspec` command allows the caller to set custom artwork resizing specifications. These are used during a media scan to pre-cache artwork in the given size and format. The name is optional, but allows to recognize, which client would have registered a specification.
+
+Example:
+
+```
+artworkspec add 300x300_p My%20Favorite%20Controller%20App
+```
+***
+
+## logging
+`logging <group:logging group> [<persist:1>]`
+
+The `logging` command allows setting some logging levels. Today you can only set one of the following logging groups: server, radio, transcoding, scanner. The optional persist parameter defines whether the change should be persistent or not.
+
+Examples:
+
+```
+Request: "logging group:scanner<LF>"
+Response: "logging group:scanner<LF>"
+```
+***
+
+## getstring
+`getstring <STRINGTOKEN1[,STRINGTOKEN2...]>`
+
+The `getstring` command allows the caller to query one or several localized strings. String tokens can be passed as a single, concatenated value.
+
+Examples:
+
+```
+Request: "getstring HOME <LF>"
+Response: "getstring HOME:Startseite <LF>"
+
+Request: "getstring SETTINGS,SCREENSAVERS,HOME <LF>"
+Response: "getstring SETTINGS:Einstellungen SCREENSAVERS:Bildschirmschoner HOME:Startseite <LF>"
+```
+***
+
+## debug
+`debug <debug category> <OFF|FATAL|ERROR|WARN|INFO|DEBUG|?|>`
+
+The `debug` command allows the caller to query or set the server's internal debugging categories. 
+
+* Use 'OFF' to silence,
+* 'FATAL' for only seeing fatal errors,
+* 'ERROR' for non-fatal errors, etc.
+* Finally, using ? will query the current level for the category. 
+
+Valid categories can be found under Settings/Advanced/Logging.
+
+Examples:
+
+```
+Request: "debug d_files ?<LF>"
+Response: "debug d_files 0<LF>"
+
+Request: "debug d_itunes 0<LF>"
+Response: "debug d_itunes 0<LF>"
+
+Request: "debug d_stream 1<LF>"
+Response: "debug d_stream 1<LF>"
+
+Request: "debug d_stream<LF>"
+Response: "debug d_stream 0<LF>"
+```
+***
+
+## exit
+`exit`
+
+The `exit` command closes the TCP connection with the server and terminates the Command Line Interface session.
+
+Example:
+
+```
+Request: "exit<LF>"
+Response: "exit<LF>"
+(Connection terminated)
+```
+
+## stopserver
+`stopserver`
+
+The `stopserver` command shuts down the server.
+
+Example:
+```
+Request: "stopserver<LF>"
+Response: "stopserver<LF>"
+(Connection terminated)
+```
+
+## retartserver

```

Comment:
Typo :)

Reply:
What is protocol we have wrt this sort of error? 

Should I 
#1 - withdraw Pull request, fix error and re-present
or
#2- let this one go thru, raise a separate request later to fix this

please advise, keen follow the house rules, so to speak

Reply:
You can update your PR! Just create another commit and push it to the same branch. The PR will automatically be updated.

Reply:
Thanks. I pressed the buttons, and it seems to have done that now. So that `restartserver` typo corrected.